src/views/Landing.jsx

```diff
@@ -6,11 +6,28 @@ import CoreFooter from "../components/CoreFooter";
 // Datas
 import coursesData from "../data/_Courses";
 
+import { useState, useEffect } from "react";
+
+
+
 function Landing() {
 
+//Google Analytics Alert Impleenting local storage
+//if the alert is saved in local storage, then consent for the current user already occurred so the alert won't pop up again
+ 
+
+function setAlerts(){
+  alert("Please know that we use Google Analytics and cookies to track user traffic for this site. By clicking ok, you consent to flatironopensource.com's Cookie and Privacy policies.")
```

Comment:
Instead of `Alert` we should use daisyUI modal.
https://react.daisyui.com/?path=/story/actions-modal--default